libs/styles/src/lib/scss/components/_notification.scss

```diff
@@ -65,15 +69,20 @@
 
     #{$base}__content-icon {
       fill: get-color(black);
-      margin-top: 1px;
+      margin-top: 4px;
       margin-right: -6px;
     }
 
     #{$base}__close-button {
-      position: absolute;
-      top: 20px;
-      right: 20px;
+      display: flex;
       cursor: pointer;
+
+      &.compact {
```

Comment:
No need to add a modifier class for this button specifically. You can replace it with `#{$base}--size-compact #{$base}__close-button` instead.

Reply:
So if compact and regular variants have different stylings would the css look something like this?
```css
#{$base}__close-button {
// some regular style
    }

#{$base}--size-compact #{$base}__close-button {
// some compact style
    }
```

Reply:
@Seeru-crypto yes, exactly!